docs/specification/new-spec/light-client.md

```diff
@@ -0,0 +1,111 @@
+# Light client
+
+A light client is a process that connects to the Tendermint Full Node(s) and then tries to verify the Merkle proofs 
+about the blockchain application. In this document we describe mechanisms that ensures that the Tendermint light client 
+has the same level of security as Full Node processes (without being itself a Full Node). 
+
+To be able to validate a Merkle proof, a light client needs to validate the blockchain header that contains the root app hash. 
+Validating a blockchain header in Tendermint consists in verifying that the header is committed (signed) by >2/3 of the 
+voting power of the corresponding validator set. As the validator set is a dynamic set (it is changing), one of the 
+core functionality of the light client is updating the current validator set, that is then used to verify the 
+blockchain header, and further the corresponding Merkle proofs. 
+
+For the purpose of this light client specification, we assume that the Tendermint Full Node exposes the following functions over
+Tendermint RPC:
+
+```golang
+Header(height int64) (SignedHeader, error) // returns signed header for the given height
+Validators(height int64) (ResultValidators, error) // returns validator set for the given height
+LastHeader(valSetNumber int64) (SignedHeader, error)   // returns last header signed by the validator set with the given validator set number
+
+type SignedHeader struct {
+    Header Header 
+    Commit Commit
+    ValSetNumber int64 
+}
+
+type ResultValidators struct {
+    BlockHeight int64              
+    Validators  []Validator 
+    ValSetTime int64               // time the current validator set is initialised, i.e, time of the last validator change before header BlockHeight 
+}
+```
+
+We assume that Tendermint keeps track of the validator set changes and that each time a validator set is changed it is 
+being assigned the next sequence number. We can this number the validator set sequence number. Tendermint also remembers 
```

Comment:
`We can this number the validator set sequence number.` is very funky, probably needs to be grammatically corrected.

Reply:
It's a typo. It should be " We call this number the validator set sequence number".

Reply:
"We can call this number ..."